src/main/java/org/logstash/beats/Protocol.java

```diff
@@ -0,0 +1,22 @@
+package org.logstash.beats;
+
+/**
+ * Created by ph on 2016-05-16.
+ */
+public class Protocol {
+    public static final byte VERSION_1 = '1';
```

Comment:
you could create an enum out of the protocols, so you dont have to handle bytes but protocol versions instead